src/Microsoft.DocAsCode.Build.TableOfContents/TocResolver.cs

```diff
@@ -59,56 +106,91 @@ private TocItemInfo ResolveItemCore(TocItemInfo wrapper, Stack<FileAndType> stac
                         {
                             item.Items[i] = ResolveItem(new TocItemInfo(file, item.Items[i]), stack).Content;
                         }
-                        if (string.IsNullOrEmpty(item.Homepage) && string.IsNullOrEmpty(item.HomepageUid))
+                        if (string.IsNullOrEmpty(item.TopicHref) && string.IsNullOrEmpty(item.TopicUid))
                         {
                             var defaultItem = GetDefaultHomepageItem(item);
                             if (defaultItem != null)
                             {
-                                item.Homepage = defaultItem.Href;
-                                item.HomepageUid = defaultItem.Uid;
+                                item.AggregatedHref = defaultItem.Href;
+                                item.AggregatedUid = defaultItem.TopicUid;
                             }
                         }
                     }
-
+                    // for backward compatibility
+                    if (item.Href == null && item.Homepage == null)
+                    {
+                        item.Href = item.TocHref;
+                        item.Homepage = item.TopicHref;
+                    }
```

Comment:
Keep the backward compatibility code here, as homepage is only consumed in this case, and in other case, assign tocHref to href is wrong.